packages/cli/src/lib/tunneling/index.ts

```diff
@@ -25,17 +25,17 @@ export const flattenTunnels = (
 
 export class UnverifiedHostKeyError extends Error {
   constructor(
-      readonly tunnelOpts: TunnelOpts,
-      readonly hostKeySignature: string,
+    readonly tunnelOpts: TunnelOpts,
```

Comment:
can you undo formatting-only changes in unrelated code? better to turn off prettify for this project if you have it on.